README.md

```diff
@@ -107,7 +107,8 @@ Running tests
 ---
 
 ```lua
-lua tests/runner.lua
+tarantoolctl rocks make -- optionally
+tarantool tests/runner.lua
```

Comment:
Is it shell or lua? Code highlight and a comment is lua, but commands are for shell.